src/runtime/runtime.cs

```diff
@@ -1287,199 +1332,193 @@ internal static bool PyFloat_Check(IntPtr ob)
         /// Return value: New reference.
         /// Create a Python integer from the pointer p. The pointer value can be retrieved from the resulting value using PyLong_AsVoidPtr().
         /// </summary>
-        [DllImport(_PythonDll, CallingConvention = CallingConvention.Cdecl)]
-        internal static extern IntPtr PyLong_FromVoidPtr(IntPtr p);
+        internal static NewReference PyLong_FromVoidPtr(IntPtr p) => Delegates.PyLong_FromVoidPtr(p);
 
         /// <summary>
         /// Convert a Python integer pylong to a C void pointer. If pylong cannot be converted, an OverflowError will be raised. This is only assured to produce a usable void pointer for values created with PyLong_FromVoidPtr().
         /// </summary>
-        [DllImport(_PythonDll, CallingConvention = CallingConvention.Cdecl)]
-        internal static extern IntPtr PyLong_AsVoidPtr(IntPtr ob);
+        
+        internal static IntPtr PyLong_AsVoidPtr(BorrowedReference ob) => Delegates.PyLong_AsVoidPtr(ob);
 
-        [DllImport(_PythonDll, CallingConvention = CallingConvention.Cdecl)]
-        internal static extern IntPtr PyFloat_FromDouble(double value);
+        
+        internal static IntPtr PyFloat_FromDouble(double value) => Delegates.PyFloat_FromDouble(value);
 
-        [DllImport(_PythonDll, CallingConvention = CallingConvention.Cdecl)]
-        internal static extern IntPtr PyFloat_FromString(IntPtr value, IntPtr junk);
+        
+        internal static IntPtr PyFloat_FromString(IntPtr value, IntPtr junk) => Delegates.PyFloat_FromString(value, junk);
```

Comment:
Is this in use? The API docs don't say anything about this ominous `junk` parameter.

Reply:
I did not really walk through the APIs to check them apart from string usages. This looks bogus. I will change the signature.